src/js/lib.js

```diff
@@ -304,7 +301,7 @@ var isEmpty = function(obj) {
  * @private
  */
 var hasClass = function(element, classToCheck){
-  return ((' ' + element.className + ' ').indexOf(' ' + classToCheck + ' ') !== -1);
+  return ((` ${element.className} `).indexOf(` ${classToCheck} `) !== -1);
```

Comment:
I know they keep spacing but I don't it's as clear that we're adding spacing for a reason when you read it in template syntax.